README.md

```diff
@@ -64,11 +64,14 @@ And call the shelf into you prefered rest route
 You can see the full functionality into the [TODO-LIST ON HERBS repository](https://github.com/herbsjs/todolist-on-herbs)
 
 
+TIP: If you need to block the shelf renderization, use process.env.HERBS_SHELF_RENDER = "false". This will render a unauthorized message instead the usecase list. Recommended for production environments. By default, the renderization is allowed.
```

Comment:
I believe it isn't a "TIP", it's a feature, so we can write this feature into a "production session", "Envs" or into "config session", something like this